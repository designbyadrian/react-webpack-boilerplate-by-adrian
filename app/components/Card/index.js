import React from 'react';
import { Link } from 'react-router-dom';

import './style.sass';

const Card = props => (
  <div className="card" styleName="card">
    <div className="card-body">
      <h5 className="card-title">{props.title}</h5>
      <Link to={`/post/${props.slug}`} className="btn btn-primary">Read more</Link>
    </div>
  </div>
);

export default Card;
