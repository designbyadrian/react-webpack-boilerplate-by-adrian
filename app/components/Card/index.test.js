import React from 'react';
import { render } from 'react-testing-library';

import Card from './index';

describe('components/Card', () => {
  it('matches snapshot', () => {
    const { container } = render(<Card />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
