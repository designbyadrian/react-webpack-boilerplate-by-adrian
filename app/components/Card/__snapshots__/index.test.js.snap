// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Card matches snapshot 1`] = `
<div
  class="card style-module-card"
>
  <div
    class="card-body"
  >
    <h5
      class="card-title"
    />
    <a
      href="/post/undefined"
    >
      readmore
    </a>
  </div>
</div>
`;
